src/ctapipe/reco/impact.py

```diff
@@ -204,29 +208,31 @@ def __call__(self, event):
             mask = event.dl1.tel[tel_id].image_mask
 
             # Dilate the images around the original cleaning to help the fit
-            mask = dilate(self.subarray.tel[tel_id].camera.geometry, mask)
-            mask = dilate(self.subarray.tel[tel_id].camera.geometry, mask)
-            mask = dilate(self.subarray.tel[tel_id].camera.geometry, mask)
+            for _ in range(3):
+                mask = dilate(self.subarray.tel[tel_id].camera.geometry, mask)
+
             mask_dict[tel_id] = mask
 
         # Next, we look for geometry and energy seeds from previously applied reconstructors.
         # Both need to be present at elast once for ImPACT to run.
```

Comment:
Should be configurable I think, not just take any?

i.e. `geometry_seed = Unicode(default="HillasReconstructor")`?